### PR TITLE
ST-5293: Override version of netty used by ZK to address CVE

### PR DESCRIPTION
**Pull request by elismaga**
Exclude version of netty used by zk and use a version that has the CVE fixed.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -89,6 +89,7 @@
         <confluent-log4j.version>1.2.17-cp2</confluent-log4j.version>
         <zkclient.version>0.11</zkclient.version>
         <zookeeper.version>3.5.9</zookeeper.version>
+        <netty.version>4.1.61</netty.version>
```
Comment (ewencp) [inline pom.xml:92]: Do we want .61 or .63? I'm not sure how netty versioning works so not sure if those are equally safe... although I suppose this does seem like the minimal update required.

**Review by ewencp — APPROVED** [review]
One question about the exact version we want, but lgtm

File: pom.xml (hunk 1 above)
Comment (elismaga) [reply pom.xml]: @ewencp changed the version to 4.1.63 since that is the version schema registry is already using.

Comment (ewencp) [reply pom.xml]: Sounds good, still lgtm

**Pull request merged**